src/pages/index.js

```diff
@@ -129,28 +110,100 @@ function closeModal(modal) {
   modal.classList.remove("modal_opened");
   document.removeEventListener('keydown', handleEscKey);
   modal.removeEventListener('mousedown', handleOverlayClick);
+
+  const formEl = modal.querySelector("form");
+  if (formEl) {
+    resetForms(formEl);
+    resetValidation(formEl, settings);
```

Comment:
Please, delete it from `closeModal`

There should **not** be any code in universal functions that doesn’t suit every popup.

Reply:
This makes sense, updated.